stack-guideline.md

```diff
@@ -0,0 +1,68 @@
+# Abtion stack
+###### Or Ab-*stack*-tion!  🤦‍♂️
+![abtion image](https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif)
+
+
+In Abtion we strive for the most efficient and easiest workflow.
+This docs is thought of being a docs we all can turn to if we get in doubt of how we deal with managing certain stuff, such as CI/CD.
+Also, to make it easier to on-board new employees.
+
+
+## Frontend
+As an Abtioneer we love React when working on the FE (Front-End), but we also care about Vue and is always curious of what exciting technologies that might pop-up in the future. But for now most of our FE apps is built with React.
```

Comment:
As Abtioneers we love OR As an Abtioneer I love

Reply:
and ARE always curious

Reply:
what exciting technologies might pop-up in the future. (no that)

Reply:
our FE apps ARE built with react

Reply:
@bohme Actually not all of our apps is built with react, we have some projects built with Vue. But i agree the majority is react apps. 😄

Reply:
I am aware, but we have more than one app, so it should be _are_ not _is_ ;)
`But for now most of our FE apps ARE built with React.`